### DISCUSSION TITLE
Coding question, disabling some JS

### DISCUSSION THREAD

**Discussion by ZuluPilgrim** [GetPublii/Publii#1456] [General] [answered]
I am using the default theme that comes with Publii.

When the page loads the header starts with 
`<header class="top" id="js-header">`

As I scroll up and down the page it changes to
`<header class="top is-visible" id="js-header">`

or

`<header class="top is-invisible" id="js-header">`

How or where do I disable the code that makes this change?

**Comment by bobmitro** [comment, marked as answer]
You need to modify the **script.min.js** file located at the following path: _Documents/Publii/sites/YOUR_SITE/input/themes/YOUR_THEME/assets/js/_

This is a minified version of the script.js file, so I recommend editing the script.js file first and then minifying it to the script.min.js file.

**Note:**
To preserve your custom code and prevent it being overwritten the next time the theme is updated, please ensure that you have created override files; these are, essentially, copies of the existing theme files that are stored separately from the main theme files → https://getpublii.com/dev/theme-overrides/

**Reply by ZuluPilgrim** [reply]
Perfect, that was easy :)
